Assets/HoloToolkit/Input/Scripts/Interactions/TapToPlace.cs

```diff
@@ -181,6 +195,18 @@ private void ToggleSpatialMesh()
             }
         }
 
+        private void CalculateColliderOffset()
+        {
+            Collider[] colliders = GetComponentsInChildren<Collider>();
```

Comment:
We should probably get the collider from this object as well to add into our calculation.

Reply:
GetComponentsInChildren gets components from both the current gameObject and its children anyway: https://docs.unity3d.com/ScriptReference/Component.GetComponentsInChildren.html

And isn't it useful to get the midpoint of the object and it's children?

Thanks for the feedback btw 👍

Reply:
(are you saying use Getcomponents instead of GetComponentsInChildren btw?)

Reply:
> GetComponentsInChildren gets components from both the current gameObject and its children

It does?! I learned something new today 😆

Reply:
> And isn't it useful to get the midpoint of the object and it's children?

Yes I agree, but if there's a collider on a child that's outside the collider on this object, you could run into problems. Ideally the collider on this object should encompass _all_ the child objects.

Reply:
Ah OK then, fair enough